projects/plugins/jetpack/extensions/blocks/podcast-player/icons/link.js

```diff
@@ -1,10 +1,7 @@
 /* eslint-disable jsdoc/check-tag-names */
 
-import { SVG, Path } from '@wordpress/components';
+import { SVG, Path } from '@wordpress/primitives';
 
-/**
- * @todo: Replace with `@wordpress/icons` when available
```

Comment:
It would depend on the tree shaking, I think. `@wordpress/icons` [is not externalized](https://github.com/WordPress/gutenberg/blob/334dc9c854e8bf4a2758ecda652ffa027e898633/packages/dependency-extraction-webpack-plugin/lib/util.js#L2), so if tree-shaking does a good job of pulling in only the needed icon it might be worthwhile.

Reply:
Oh, my bad! Do you think it's worth keeping the comment, then?

Reply:
:shrug: I probably would, since it seems like it's still worth investigating whether the tree-shaking does the right thing. Feel free to reword it to be more explicit about what exactly would need to be investigated.

OTOH, when I tracked down the PR that added the non-externalizing config, I see there was a lot of other discussion and you were involved. So I think you'll know better than I would about all of it. 😀

Reply:
I'll leave it without the comment, I think, as there isn't a huge benefit to reusing the icon in this case, and introducing the dependency could come with caveats. Thanks!